app/assets/javascripts/modules/remove-filter.js

```diff
@@ -55,38 +54,37 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
          * Just removing " beta " from the haystack would result in
          * "alphagamma", which is wrong.
          */
-        var haystack = ' ' + currentVal.trim() + ' ';
-        var needle = ' ' + decodeEntities(removeFilterValue.toString()) + ' ';
-        var newVal = haystack.replace(needle, ' ').replace(/  /g, ' ').trim();
+        var haystack = ' ' + currentVal.trim() + ' '
+        var needle = ' ' + decodeEntities(removeFilterValue.toString()) + ' '
+        var newVal = haystack.replace(needle, ' ').replace(/ {2}/g, ' ').trim()
```

Comment:
Can you explain this change please?

Reply:
Again, this was a `standard --fix` change. I guess they think it's better to use a regex that counts 2 spaces, rather than to use two spaces itself.

Seems odd that we're doing a 2-space-to-1-space replacement in the first place 🤔 but out of scope of this PR!